docs/pages/upcoming-releases.mdx

```diff
@@ -6,114 +6,91 @@ tags:
  - platform-wide
 ---
 
-The Teleport team delivers a new major release roughly every 3 months.
+Teleport releases a new major version once per year, and provides
+security-critical support for the current and previous major version. We
+support each major version for 24 months.
 
-## Teleport
-
-### Teleport 12.3.0
-
-####  Automatic user creation for desktop access
-
-Teleport's passwordless access for local users will provide the ability to
-create Windows users on demand, similar to how host-user creation works for SSH
-servers.
-
-#### SFTP UX enhancements
-
-Will include fixes for wildcard patterns, excessive audit logging and PyCharm
-compatibility. [#22748](https://github.com/gravitational/teleport/issues/22748)
-[#21518](https://github.com/gravitational/teleport/issues/21518)
-[#22982](https://github.com/gravitational/teleport/issues/22982)
-[#22263](https://github.com/gravitational/teleport/issues/22263)
-[#20863](https://github.com/gravitational/teleport/issues/20863)
-[#23593](https://github.com/gravitational/teleport/issues/23593)
-
-| Version | Date          |
-|---------|---------------|
-| 12.3.0  | May 1st, 2023 |
+The most recent major version of Teleport, referred to as the _Current Version_,
```

Comment:
```suggestion
The most recent major version of Teleport, referred to as the **current version**,
```
Since this is still a common noun. I think we also standardize on bold for keywords.